### DISCUSSION TITLE
How to generate offline documentation

### DISCUSSION THREAD

**Discussion by Momentum-TN** [pyg-team/pytorch_geometric#2389] [Q&A]
Hey @rusty1s ,

Thank you for this amazing framework ♥!

My question is: how to generate offline documentation ?

PS: I am using windows 10

**Comment by rusty1s** [comment]
In case you have `torch_geometric` installed on your system, you can download the PyG source code and run the following:
```
cd docs
pip install sphinx sphinx_rtd_theme
make html
```
The docs are located in `build/html/index.html`.

**Reply by wsad1** [reply]
I think it would be helpful to add this to the contributions.md. I can create a PR if you feel it'll be useful.

**Reply by rusty1s** [reply]
Sounds great :) Thanks!
